docs/command-reference.md

```diff
@@ -358,7 +358,7 @@ Ulimit flags:
 
 Verify flags:
 
-- :nerd_face: `--verify`: Verify the image (none|cosign). See [`docs/cosign.md`](./cosign.md) for details.
+- :nerd_face: `--verify`: Verify the image (none|cosign|notation). See [`docs/cosign.md`](./cosign.md) and [`docs/notation.md`](./notation.md) for details.
```

Comment:
can we change `docs/cosign.md` to `./cosign.md` (same for `notation.md`). It's a relative path to the repo root but this doc is already in `docs` dir.